lib/validate_encrypt_utils.pm

```diff
@@ -230,4 +231,13 @@ sub verify_locked_encrypted_partition {
     record_info('lock OK', "Encrypted partition '/dev/$enc_disk_part' still locked");
 }
 
+sub validate_activation_encrypted_partition {
+    my ($mapped_device, $message, $properties) = @_;
```

Comment:
for more than one param as agreed, let's use named arguments ( mapped_device => x, message => y, etc.) in the call.